tests/test_macros_oval.py

```diff
@@ -0,0 +1,1042 @@
+            parameter='speed',
+            value='100',
+            missing_parameter_pass=false,
+            application='',
+            multi_value=false,
+            missing_config_file_fail=false,
+        ) }}}""",
+        """[vehicle]
+        speed = 200""",
+        "false"
+    )
+    tester.test(
+        "INI wrong value which is a substring",
+        r"""{{{ oval_check_ini_file(
+            path='CONFIG_FILE',
+            section="vehicle",
+            parameter='speed',
+            value='100',
+            missing_parameter_pass=false,
+            application='',
+            multi_value=false,
+            missing_config_file_fail=false,
+        ) }}}""",
+        """[vehicle]
+        speed = 10000""",
+        "false"
+    )
+    tester.test(
+        "INI overwritten",
+        r"""{{{ oval_check_ini_file(
+            path='CONFIG_FILE',
+            section="vehicle",
+            parameter='speed',
+            value='100',
+            missing_parameter_pass=false,
+            application='',
+            multi_value=false,
+            missing_config_file_fail=false,
+        ) }}}""",
+        """[vehicle]
+        speed = 100
+        speed = 200""",
+        "false"
+    )
+    tester.test(
+        "INI correct value in a wrong section",
+        r"""{{{ oval_check_ini_file(
+            path='CONFIG_FILE',
+            section="vehicle",
+            parameter='speed',
+            value='100',
+            missing_parameter_pass=false,
+            application='',
+            multi_value=false,
+            missing_config_file_fail=false,
+        ) }}}""",
+        """[house]
+        speed = 100""",
+        "false"
+    )
+    tester.test(
+        "INI section overwritten",
+        r"""{{{ oval_check_ini_file(
+            path='CONFIG_FILE',
+            section="vehicle",
+            parameter='speed',
+            value='100',
+            missing_parameter_pass=false,
+            application='',
+            multi_value=false,
+            missing_config_file_fail=false,
+        ) }}}""",
+        "[vehicle]\n[house]\nspeed = 100",
+        "false"
+    )
+    tester.test(
+        "INI no section at all",
+        r"""{{{ oval_check_ini_file(
+            path='CONFIG_FILE',
+            section="vehicle",
+            parameter='speed',
+            value='100',
+            missing_parameter_pass=false,
+            application='',
+            multi_value=false,
+            missing_config_file_fail=false,
+        ) }}}""",
+        "speed = 100",
+        "false"
+    )
+    tester.test(
+        "INI extra newlines",
+        r"""{{{ oval_check_ini_file(
+            path='CONFIG_FILE',
+            section="vehicle",
+            parameter='speed',
+            value='100',
+            missing_parameter_pass=false,
+            application='',
+            multi_value=false,
+            missing_config_file_fail=false,
+        ) }}}""",
+        "[vehicle]\nspeed =\n100",
```

Comment:
You can use [dedent](https://docs.python.org/3.6/library/textwrap.html#textwrap.dedent) to preserve nice input while removing leading whitespace.